aspnetcore/web-api/define-controller/samples/WebApiSample.Api.22/Controllers/ProductsController.cs

```diff
@@ -52,7 +53,7 @@ public async Task<ActionResult<Product>> GetByIdAsync(int id)
         }
 
         [HttpPost]
-        [ProducesResponseType(201)]
+        [ProducesResponseType(typeof(Product), 201)]
```

Comment:
The type is inferred from the `T` in `ActionResult<T>`.

```suggestion
        [ProducesResponseType(201)]
```